src/test/java/htmlpublisher/HtmlPublisherIntegrationTest.java

```diff
@@ -85,6 +90,35 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
         assertFalse(tab2Files.contains("dummy.html"));
     }
 
+    @Test
+    public void testIncludeSameReport() throws Exception {
```

Comment:
Can you please include a test for both the false & true settings of the flag just to ensure coverage of both scenarios?

Reply:
test added